cmd/serve/main.go

```diff
@@ -0,0 +1,94 @@
+package serve
+
+import (
+	"github.com/aliazizii/IMDB-In-Golang/config"
+	"github.com/aliazizii/IMDB-In-Golang/internal/auth"
+	"github.com/aliazizii/IMDB-In-Golang/internal/database"
+	"github.com/aliazizii/IMDB-In-Golang/internal/handler"
+	"github.com/aliazizii/IMDB-In-Golang/internal/store/comment"
+	"github.com/aliazizii/IMDB-In-Golang/internal/store/movie"
+	"github.com/aliazizii/IMDB-In-Golang/internal/store/user"
+	"github.com/aliazizii/IMDB-In-Golang/internal/store/vote"
+	"github.com/golang-jwt/jwt/v5"
+	echojwt "github.com/labstack/echo-jwt/v4"
+	"github.com/labstack/echo/v4"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func main(cfg *config.Config) {
+	app := echo.New()
+
+	db, err := database.New(cfg.DB)
+	if err != nil {
+		logrus.Fatal(err)
```

Comment:
It's better to add additional context like

```go
logrus.Fatalf("database connection failed: %s", err)
```
to places where we log the error. It'll make finding the error stack easier.